snow/engine/snowman/transitive_test.go

```diff
@@ -2885,37 +2866,21 @@ func TestEngineBubbleVotesThroughInvalidBlock(t *testing.T) {
 	*queried = false
 	// Prepare to PushQuery [blk2] after receiving a Gossip message with [blk2].
 	sender.SendPushQueryF = func(_ context.Context, inVdrs set.Set[ids.NodeID], requestID uint32, blkBytes []byte) {
-		if *queried {
-			t.Fatalf("Asked multiple times")
-		}
+		require.False(*queried, "Asked multiple times")
 		*queried = true
 		*queryRequestID = requestID
 		vdrSet := set.Set[ids.NodeID]{}
 		vdrSet.Add(vdr)
-		if !inVdrs.Equals(vdrSet) {
-			t.Fatalf("Asking wrong validator for preference")
-		}
-		if !bytes.Equal(blk2.Bytes(), blkBytes) {
-			t.Fatalf("Asking for wrong block")
-		}
+		require.Equal(inVdrs, vdrSet, "Asking wrong validator for preference")
+		require.Equal(blk2.Bytes(), blkBytes, "Asking for wrong block")
 	}
 	// Expect that the Engine will send a PushQuery after receiving this Gossip message for [blk2].
-	if err := te.Put(context.Background(), vdr, constants.GossipMsgRequestID, blk2.Bytes()); err != nil {
-		t.Fatal(err)
-	}
-
-	if !*queried {
-		t.Fatalf("Didn't ask for preferences regarding blk2")
-	}
+	require.NoError(te.Put(context.Background(), vdr, constants.GossipMsgRequestID, blk2.Bytes()))
+	require.True(*queried, "Didn't ask for preferences regarding blk2")
 
 	// After a single vote for [blk2], it should be marked as accepted.
-	if err := te.Chits(context.Background(), vdr, *queryRequestID, []ids.ID{blk2.ID()}, nil); err != nil {
-		t.Fatal(err)
-	}
-
-	if blk2.Status() != choices.Accepted {
-		t.Fatalf("Expected blk2 to be Accepted, but found status: %s", blk2.Status())
-	}
+	require.NoError(te.Chits(context.Background(), vdr, *queryRequestID, []ids.ID{blk2.ID()}, nil))
+	require.Equalf(blk2.Status(), choices.Accepted, "Expected blk2 to be Accepted, but found status: %s", blk2.Status())
```

Comment:
I'm fairly ambivalent on if we leave the descriptions... But we shouldn't be formatting the descriptions like this... The expected value and the actual value are already output in the error log
```suggestion
	require.Equal(choices.Accepted, blk2.Status())
```